src/components/ui/slide-thumbnail/index.tsx

```
@@ -0,0 +1,109 @@
+          >
+            <div className={S.previewOverlay()} aria-hidden="true" />
+            <pre className={S.previewCode()}>
+              {code || "Empty slide"}
+            </pre>
+          </div>
+
+          <div className={S.meta()}>
+            <span className={S.title()} title={title || "Untitled"}>
+              {title || "Untitled"}
+            </span>
+            <span className={S.duration()} aria-label={`Duration ${duration} seconds`}>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)